### ISSUE TITLE
Allow WCAG Accessibility URL to be set in management pages

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#979]
Allow WCAG Accessibility link URL to be managed from management pages. You can set a different URL as the author on individual projects but to change for all projects you have to edit colourChanger.html / application.js directly which can easily be lost on upgrade

**Comment by ronm123** [comment]
Just to add to this I've also been asked the following:
whether it's possible to change the logo in each case e.g. projects and workspace.
whether where we  have the text link to the accessibility guide in colour changer should be followed by a text link to the accessibility statement in addition to the logo link we now have.

Any options added to management should probably reflect the options available at project level so logo, link, text, alt/tooltip and for projects as well as workspace.

**Comment by FayCross** [comment]
Closing as it's part of the larger issue listing new options that could be controlled via management
